Tasks/UniversalPackagesV0/universalpublish.ts

```diff
@@ -141,7 +111,45 @@ export async function run(artifactToolPath: string): Promise<void> {
             packageVersion: version,
         } as artifactToolRunner.IArtifactToolOptions;
 
-        publishPackageUsingArtifactTool(publishDir, publishOptions, toolRunnerOptions);
+        publishStatus = publishPackageUsingArtifactTool(publishDir, publishOptions, toolRunnerOptions);
+
+        //If package already exist, retry with a newer version
+        if (publishStatus != null && publishStatus == 17) {
```

Comment:
Consider using a loop instead of repeating the code. This will also allow us to vary the number of tries. (doesn't need to be configurable at runtime IMO, but makes a quick and safe change if we want to change the number of tries).

Reply:
Added the loop